### PR TITLE
Bookstore: initialize project with components 🎗️

### PR DESCRIPTION
**Pull request by ritikarawat220**
### In this PR ☁️

- [x] Setup a new **React application** using [_**Create React App**_](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- [x] Create a directory for reusable components: **_/components._**
- [x] Install [_**React Router**_](https://reactrouter.com/en/main/start/tutorial#setup)
- [x] [_**Setup your router**_](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of    _**< App>**_
- [x] Add 2 routes in the application:
  /, which contains
- [x] A _**book list**_, where each book has a _**title**_ and _**author**_
- [x] A form that allows for creating a _**new book**_

 **_/categories_**
- [x] This should display a button that checks for a **_status_**
- [x] Include a _**navigation component**_ that includes links to the 2 routes

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @ritikarawat220 , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

- [x] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [x] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [x] Please remove the Microverse logo from your README file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,224 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
```
Comment (Ismailco) [inline README.md]: - [ ] The link to the issue page is broken kindly make sure to update it so it redirects to the issues page instead

File: package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```
Comment (Ismailco) [inline package.json:5]: - [ ] Kindly make sure to add `react-router-dome` to your project so you can make the routes in this project

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ritikarawat220, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ use CRA
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @ritikarawat220 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## HIghlights
- No linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**